Testing/SystemTests/scripts/systestrunner.py

```diff
@@ -0,0 +1,133 @@
+
+DEFAULT_QT_API = "pyqt5"
+THIS_MODULE_DIR = os.path.dirname(os.path.realpath(__file__))
+DEFAULT_FRAMEWORK_LOC = os.path.realpath(os.path.join(THIS_MODULE_DIR, "..", "lib", "systemtests"))
+SANS_TEST_LOC = os.path.realpath(os.path.join(THIS_MODULE_DIR, "..", "tests", "framework", "ISIS", "SANS"))
+DATA_DIRS_LIST_PATH = os.path.join(THIS_MODULE_DIR, "datasearch-directories.txt")
+SAVE_DIR_LIST_PATH = os.path.join(THIS_MODULE_DIR, "defaultsave-directory.txt")
+DEFAULT_LOG_LEVEL = "information"
+DEFAULT_ARCHIVE_SEARCH = True
+DEFAULT_MAKE_PROP = True
+DEFAULT_EXECUTABLE = sys.executable
+
+
+def kill_children(processes):
+    for process in processes:
+        process.terminate()
+    # Exit with status 1 - NOT OK
+    sys.exit(1)
+
+
+def main(argv):
+    # Set the Qt version to use during the system tests
+    os.environ["QT_API"] = DEFAULT_QT_API
+
+    # import the system testing framework
+    sys.path.append(DEFAULT_FRAMEWORK_LOC)
+    import systemtesting
+
+    # allow PythonInterface/test to be discoverable
+    sys.path.append(systemtesting.FRAMEWORK_PYTHONINTERFACE_TEST_DIR)
+
+    sys.path.append(SANS_TEST_LOC)
+    #########################################################################
+    # Configure mantid
+    #########################################################################
+
+    # Parse files containing the search and save directories
+    with open(DATA_DIRS_LIST_PATH, "r") as f_handle:
+        data_paths = f_handle.read().strip()
+
+    with open(SAVE_DIR_LIST_PATH, "r") as f_handle:
+        save_dir = f_handle.read().strip()
+
+    # Configure properties file
+    mtdconf = systemtesting.MantidFrameworkConfig(
+        loglevel=DEFAULT_LOG_LEVEL, data_dirs=data_paths, save_dir=save_dir, archivesearch=DEFAULT_ARCHIVE_SEARCH
+    )
+    if DEFAULT_MAKE_PROP:
+        mtdconf.config()
+
+    #########################################################################
+    # Generate list of tests
+    #########################################################################
+
+    path_to_test = argv[1]
+    test_dir_name = os.path.dirname(path_to_test)
+    test_file_name = os.path.basename(path_to_test)
+    test_module_name = os.path.splitext(test_file_name)[0]
+    test_spec = importlib.util.spec_from_file_location(test_module_name, path_to_test)
+    test_module = importlib.util.module_from_spec(test_spec)
+    test_spec.loader.exec_module(test_module)
+    test_classes = dict(inspect.getmembers(test_module, inspect.isclass))
+
+    test_class_names = [test_class for test_class in test_classes if
+                    systemtesting.isValidTestClass(test_classes[test_class]) and test_class != "MantidSystemTest"]
+
+    if not test_class_names:
+        raise NameError(f"No test classes found in system test module: {test_module_name}.")
+
+    runner = systemtesting.TestRunner(
+        executable=DEFAULT_EXECUTABLE,
+        # see InstallerTests.py for why lstrip is required
+        exec_args="--classic",
```

Comment:
```suggestion
        executable=DEFAULT_EXECUTABLE,
        exec_args="--classic",
```
comment maybe not needed anymore?

Reply:
Nope, not anymore. Good catch.